INN/umbrella-ipbs
# Fix nav menu dropdown menus

## Changes

This pull request makes the following changes:

- Adds `nav` class to the `bottom-nav` main menu so that the secondary menu dropdowns work

## Why



For #35

## Testing/Questions

Features that this PR affects:

- Header menus


Questions that need to be answered before merging:


Steps to test this PR:

1. Add a secondary menu item to a parent menu item for whatever menu is set to the `Main Navigation Bottom` location
2. View on frontend and make sure that secondary menu dropdown works as expected
